src/styles/text.tsx

```diff
@@ -25,7 +25,7 @@ export const H2 = styled.h2<{ $color?: string; $fontWeight?: number | string }>`
 
 export const H3 = styled.h3<{ $color?: string; $fontWeight?: number | string }>`
   ${openSans.style}
-  font-size: 1.5rem;
+  font-size: 1.25rem;
```

Comment:
Let's check with @kyrenetam about the style system. We are using `H3` elsewhere too. 
We may need to create a custom style.

Reply:
Yeah for now let's add it as a custom style, that way we don't modify the styling elsewhere.